plugins/slicer/MONAILabel/Resources/UI/MONAILabel.ui

```diff
@@ -346,6 +333,32 @@
         </property>
        </widget>
       </item>
+      <item row="1" column="1">
+       <widget class="QComboBox" name="segmentationModelSelector">
```

Comment:
Do you need this as generic or applicable only for auto segmentation mask.. how about mask from deepedit..
If it is generic and independent.. you can add this under tools section instead of auto segmentation section

Reply:
Agree!